common/tracing/src/logging.rs

```diff
@@ -32,31 +28,16 @@ use tracing_subscriber::prelude::*;
 use tracing_subscriber::registry::Registry;
 use tracing_subscriber::EnvFilter;
 
-use crate::tracing::subscriber::DefaultGuard;
-
-/// Write logs to stdout.
-pub fn init_default_tracing(app_name: &str) {
-    static START: Once = Once::new();
-
-    START.call_once(|| {
-        init_tracing_stdout(app_name);
-    });
-}
-
 /// Init tracing for unittest.
 /// Write logs to file `unittest`.
 pub fn init_default_ut_tracing() {
     static START: Once = Once::new();
 
     START.call_once(|| {
-        let mut g = GLOBAL_UT_LOG_GUARD.as_ref().lock().unwrap();
-        *g = Some(init_global_tracing("unittest", "_logs", "DEBUG"));
+        let _ = init_global_tracing("unittest", "_logs", "DEBUG");
```

Comment:
IMHO, the returned file guards should be kept otherwise the log file will be removed at once?

Reply:
Fixed, thank you!